firmware/application/freqman.hpp

```diff
@@ -45,14 +46,61 @@ enum freqman_error {
 
 enum freqman_entry_type {
 	SINGLE = 0,
-	RANGE
+	RANGE,
+	ERROR							
 };
 
+enum freqman_entry_step {
```

Comment:
Worth adding EU airband spacing of 8.33KHz? Ref: https://en.wikipedia.org/wiki/Airband#Channel_spacing

Reply:
I have to check how to do it because I use portapack standard AM bandwidth
The step is easy , bandwidth is another story, I write it on my agenda but I have a lot of work

Reply:
Reasonable. 
Maybe when the PR gets merged I will raise it as an enhancement issue.